src/test/ui/consts/const-eval/simd/insert_extract.stderr

```diff
@@ -0,0 +1,288 @@
+warning: skipping const checks
```

Comment:
It would be cool to give these warnings a name so that they can be allowed; they happen all over the place in many tests.

Reply:
There's a PR cleaning them up: https://github.com/rust-lang/rust/pull/64470